docs/user/hbase/install.rst

```diff
@@ -114,4 +114,20 @@ The HBase data store requires the configuration file ``hbase-site.xml`` to be on
 be accomplished by placing the file in ``geoserver/WEB-INF/classes`` (you should make the directory if it
 doesn't exist).
 
+.. note::
```

Comment:
Yeah. How about a separate section at the end of this page called "Configuration"?